lib/Bio/EnsEMBL/DataCheck/Checks/CompareOntologyTerm.pm

```diff
@@ -44,10 +44,15 @@ sub tests {
   # for that here.
   my $old_dba = $self->get_old_dba();
 
-  my $desc = 'Term count has not decreased in '.
+  my $desc = 'Term counts by NAME:namespace have not decreased in '.
              $self->dba->dbc->dbname.' compared to '.$old_dba->dbc->dbname;
-  my $sql  = 'SELECT COUNT(*) FROM term';
-  row_totals($self->dba, $old_dba, $sql, undef, 1.00, $desc);
+  my $sql  = q/
+    SELECT CONCAT(ontology.name, ':', ontology.namespace), COUNT(*)
+    FROM term
+    INNER JOIN ontology ON term.ontology_id=ontology.ontology_id
+    GROUP BY ontology.ontology_id
+  /;
+  row_subtotals($self->dba, $old_dba, $sql, undef, 1.00, $desc);
```

Comment:
@james-monkeyshines is the 1.00 meaning "I will fail if the difference is 100%"

Reply:
From `row_subtotals()` docs:
```
row_subtotals($dbc1, $dbc2, $sql1, $sql2, $min_proportion, $test_name)

By default, the test only passes if the counts are exactly the same. To allow
for some wiggle room, C<$min_proportion> can be used to define the minimum
acceptable difference between the counts. For example, a value of 0.75 means
that the count for C<$dbc1> must not be less than 75% of the count for C<$dbc2>.
If two SQL statements are given, a value of 0.75 means that the count for
C<$sql1> must not be less than 75% of the count for C<$sql2>.
```

Reply:
:+1: